src/game/camera.c

```diff
@@ -1116,13 +1141,98 @@ s32 snap_to_45_degrees(s16 angle) {
     return angle;
 }
 
+#ifdef EIGHT_DIR_CAMERA_COLLISION
+void eight_dir_collision_handler(struct Camera *c) {
```

Comment:
would like some lines of spacing between parts of the code in this function and the below function, its a little hard to read

Reply:
mark it as resolved if you think it's ok now